extensions/rust/tests/benchmark_pymatgen.py

```
@@ -0,0 +1,186 @@
+    """Load JSON structures from matterviz."""
+    structures = {}
+    if not MATTERVIZ_STRUCTURES_DIR.exists():
+        return structures
+
+    for json_file in MATTERVIZ_STRUCTURES_DIR.glob("*.json"):
+        try:
+            data = json.loads(json_file.read_text())
+            if data.get("@class") == "Structure":
+                structures[json_file.stem] = Structure.from_dict(data)
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)